config.yaml

```diff
@@ -1,28 +1,133 @@
 stages:
-  # - make_assets
+  - make_assets
+  - prepare_view
   - embed_reveal
   - docusaurus
 
 build_dir: /build
 output_type: last
 output_dir: /output
 
+lab_structure:
+  - title: Lab 1 - Number Representation
+    filename: lab1.md
+    content:
+      - reading/intro-computer-architecture_binary-hex_reading_README.md
```

Comment:
Since all the .md files in each reading folder were named README.md, I had to rename them in order to group them in .view/reading . However, in my most recent commit, I renamed them to the suggested format.